roles/engineer.md

```diff
@@ -50,24 +50,19 @@ We need someone:
  - to regularly improve our process for delivering software
  - to ensure the done done completion of tasks
 
-### +Lead
+### +Mentor
 
 We need someone:
 
-- to foster good engineering standards on a project
-- to be the primary day-to-day client liaison on technology-lead deliveries
-- to mentor developers on their team
-- to drive process improvements on their team
+ - to provide a mentorship role to a colleague
+ - to encourage learning across the company
 
-We expect someone in the role:
+## Engineer Leadership
 
-- to need less than 1 day per week of Technical Director involvement in their deliveries
-- to be mentoring middleweight developers on their team within 2 months
-- to be improving our practices across the business within 3 months
+Traits recognise value that Engineers can provide customers and the company. They are hats and as Engineers develop their skills, Engineers will begin to wear more of those hats. Leadership is an emergent quality that develops under individual traits. The longer an Engineer wears a trait the more of a leader in that area they will become.
 
-### +Mentor
+Engineer Leadership is a property that emerges when an individual becomes a leader across multiple traits and starts to perform more vital roles within engagements with customers.
 
-We need someone:
+For example, when an Engineer becomes a leader in +Customer and +Delivery they will naturally begin to be the primary point of contact to the customer for the engagement, not just in regards to communication and agile process but also in other commercial areas. Relationships will naturally evolve between Engineers fulfilling more skills in the team and the customer, this seniority will be recognised. As these relationships develop with customers, more commercial level responsibility will fall on this emerging leader.
```

Comment:
'will fall' or 'should be assumed'? responsibility being taken rather than given, and all that